transportable-udfs-plugin/src/main/java/com/linkedin/transport/plugin/Defaults.java

```diff
@@ -78,13 +78,15 @@ private static final String getVersion(final String platform) {
           JavaLanguageVersion.of(17),
           ImmutableList.of(
               DependencyConfiguration.builder(IMPLEMENTATION, "com.linkedin.transport:transportable-udfs-trino", TRANSPORT_VERSION).build(),
-              DependencyConfiguration.builder(COMPILE_ONLY, "io.trino:trino-main", TRINO_VERSION).exclude("com.nimbusds").build()
+              DependencyConfiguration.builder(COMPILE_ONLY, "io.trino:trino-main", TRINO_VERSION)
+                  .exclude("com.nimbusds").exclude("org.slf4j", "slf4j-api").build()
```

Comment:
Since com.nimbusds:nimbus-jose-jwt:9.14 is replaced by com.nimbusds:nimbus-jose-jwt:9.31, why do we need to exclude `com.nimbusds` still?

Reply:
I am not sure the exclusion in `build.gradle` applies here, since we cannot verify these hardcoded dependency with `./gradlew transportable-udfs-plugin:dependencies`. Maybe you can verify that with more perspective. I can remove it though since `io.trino:trino-main:406` is already imported anyways.

Reply:
Fixed.

Reply:
I was worried that if this dependency is not overridden the gradle plugin may still try to consume `9.14`; this is probably not an issue as I checked in artifactory that `com.nimbusds:nimbus-jose-jwt:9.14` is present in `ext-libraries`

Reply:
> I am not sure the override in build.gradle applies here

I think the override does not apply here for sure. This default platform defines the environment where the UDF project consuming Transport gradle plugin will be compiled in. The substitution of nimbus 9.31 only applies within this Transport repository. You can verify if this is the case.